app/decorators/service_provider_session_decorator.rb

```diff
@@ -19,18 +19,24 @@ def nav_partial
   end
 
   def registration_heading
-    sp = ActionController::Base.helpers.content_tag(:strong, sp_name)
-    ActionController::Base.helpers.safe_join(
-      [I18n.t('headings.create_account_with_sp', sp: sp).html_safe]
+    sp = view_context.content_tag(
+      :strong, t('headings.create_account_with_sp.sp_text', sp: sp_name)
+    )
+
+    view_context.safe_join(
```

Comment:
This might just be easier if we put it in the template as two lines?

```slim.erb
= content_tag :strong do
  = t('headings.create_account_with_sp.sp_text', sp: sp_name)
'
= t('headings.create_account_with_sp.app_text')
```